net-vpn/i2pd/i2pd-2.25.0.ebuild

```diff
@@ -43,7 +43,9 @@ CMAKE_USE_DIR="${S}/build"
 
 DOCS=( README.md contrib/i2pd.conf contrib/tunnels.conf )
 
-PATCHES=( "${FILESDIR}/${PN}-2.14.0-fix_installed_components.patch" )
+PATCHES=( "${FILESDIR}/${PN}-2.14.0-fix_installed_components.patch"
+	"${FILESDIR}/i2pd-2.25.0-link.patch"
```

Comment:
Does an EAPI bump actually imply all those patches? Or have you added independent fixes?